sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationLiveTests.cs

```diff
@@ -11,30 +11,32 @@
 namespace Azure.AI.FormRecognizer.DocumentAnalysis.Tests
 {
     /// <summary>
-    /// The suite of tests for the <see cref="DocumentModelAdministrationClient"/> class.
+    /// The suite of tests for the document model methods in the <see cref="DocumentModelAdministrationClient"/> class.
     /// </summary>
     /// <remarks>
     /// These tests have a dependency on live Azure services and may incur costs for the associated
     /// Azure subscription.
     /// </remarks>
     [IgnoreServiceError(400, "InvalidRequest", Message = "Content is not accessible: Invalid data URL", Reason = "https://github.com/Azure/azure-sdk-for-net/issues/28923")]
-    public class DocumentModelAdministrationClientLiveTests : DocumentAnalysisLiveTestBase
```

Comment:
Split the `DocumentModelAdministrationClientLiveTests` class into two different classes: `DocumentModelAdministrationLiveTests` (for model-related APIs) and `ManagementOperationsLiveTests` (for management-related APIs). We already had a separate `DocumentClassifierAdministrationLiveTests` for the classifier-related APIs.

Reply:
No changes were made to the tests.